lang/scala/scala.py

```diff
@@ -8,32 +8,80 @@
 
 # Scala Common Types
 scala_common_types = {
-    "Byte": "Byte",
-    "Int": "Int",
-    "Double": "Double",
-    "Short": "Short",
-    "Float": "Float",
-    "Long": "Long",
-    "Boolean": "Boolean",
-    "Char": "Char",
-    "String": "String",
-    "None": "None"
+    "boolean": "Boolean",
+    "int": "Int",
+    "float": "Float",
+    "byte": "Byte",
+    "double": "Double",
+    "long": "Long",
+    "char": "Char",
+    "unit": "Unit",
+    "any": "Any",
+    "any val": "AnyVal",
+    "string": "String",
+    "thread": "Thread",
+    "exception": "Exception",
+    "throwable": "Throwable",
+    "none": "None",
+    "success": "Success",
+    "failure": "Failure",
 }
 
 # Scala Common Generic Types
 scala_common_generic_types = {
-    "seq": "Seq",
-    "set": "Set",
-    "list": "List",
     "array": "Array",
+    "deque": "Deque",
+    "future": "Future",
+    "list": "List",
     "map": "Map",
-    "stack": "Stack"
+    "nil": "Nil",
+    "option": "Option",
+    "queue": "Queue",
+    "seq": "Seq",
```

Comment:
```suggestion
    "seek": "Seq",
```

Reply:
Good suggestion. I also modified the command for `Deque` from `deque` to `deck`.